mailpoet/assets/js/src/wizard/steps/pitch-mss-step/second-part.tsx

```diff
@@ -55,6 +55,7 @@ function MSSStepSecondPart(): JSX.Element {
             'welcomeWizardMSSSecondPartInputPlaceholder',
           )}
           isFullWidth
+          forceRevealed
```

Comment:
Just skimming through the code, but where is `forceRevealed` defined in the second-part.tsx?

Reply:
@websupporter this is equivalent of `forceRevealed={true}` but we have ESLint rule to omit `={true}`. This means that in the wizard, the API key is always shown as part of the one-time setup.

Reply:
:TIL: